### DISCUSSION TITLE
Skip domain verification

### DISCUSSION THREAD

**Discussion by darkarchon1978** [nextcloud/all-in-one#1152] [Questions] [answered]
Hi,

How can I skip domain verification after I've already started the docker command with it?

I've tried this command:

```
sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
-e SKIP_DOMAIN_VALIDATION=true \
nextcloud/all-in-one:latest
```

It says: "docker: Error response from daemon: Conflict. The container name "/nextcloud-aio-mastercontainer" is already in use by container "e1e2d3e8d08a2cf056cb54a823c70fa6f22945f301297e1afcf53c23908463ad". You have to remove (or rename) that container to be able to reuse that name.
See 'docker run --help'.
"
The AIO setup doesn't accept my domain provided by Tailscale, I can't understand, why.

Thanks,

**Comment by Zoey2936** [comment, marked as answer]
Try before recreating:
```sh
docker stop nextcloud-aio-mastercontainer
docker rm nextcloud-aio-mastercontainer
```

**Reply by darkarchon1978** [reply]
Thank you, it worked!

**Reply by freqq68** [reply]
It works for me ! thank's a lot.
